setup.py

```diff
@@ -11,7 +11,7 @@
 import sys
 import os
 
-if len(sys.argv) not in range(2, 4):
+if len(sys.argv) not in range(1, 4):
     command = ''
```

Comment:
i didn't remove the region change i was making the deployment easier by calling "python setup.py" instead of "python setup.py <cms-name> <region>" with a default name "testapi" or else remain the same